CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Use `sentryClientName` instead of `sdk.identifier` ([#1133](https://github.com/getsentry/sentry-dart/pull/1133))
```

Comment:
Not really a feature, its more like refactoring, but `identifier` is public so its technically a breaking change.
Maybe we introduce a new section `### Refactoring`?